plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAlluxioCacheFileOperations.java

```diff
@@ -68,6 +68,7 @@ protected DistributedQueryRunner createQueryRunner()
                 .put("fs.cache.enabled", "true")
                 .put("fs.cache.directories", cacheDirectory.toAbsolutePath().toString())
                 .put("fs.cache.max-sizes", "100MB")
+                .put("fs.cache.denylist", "uncacheable")
```

Comment:
I'm not super convinced for this name.

Maybe something like `fs.cache.skip-paths` ?

`denylist` tells nothing, whether it applies to table names, files, directories or something else.